### PR TITLE
Starting documentation of the Test compiler

### PR DESCRIPTION
**Pull request by mgaudet**
Document how to build and run the test compiler.

[ci skip] Documentation change.

Issue #221
Signed-off-by: Matthew Gaudet <magaudet@ca.ibm.com>

### CODE REVIEW

File: fvtest/compilertest/README.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
+Test Compiler
+=============
+
+This file contains the OMR 'test compiler', a framework for building language
```
Comment (0xdaryl) [inline fvtest/compilertest/README.md]: * 'OMR' should be 'Eclipse OMR'
* 'test compiler' should be 'Test compiler'

File: fvtest/compilertest/README.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+
+In the case where `guess-platform.sh` is incorrect, you can set the `PLATFORM`
+environment variables to one of the supported ones, listed in
+`compilertest/build/platforms/host.mk`. 
+
+
+Running the Test Compiler
+-------------------------
+
+The test compiler has been written using [Google Test][gtest], and as such,
```
Comment (0xdaryl) [inline fvtest/compilertest/README.md]: * test -> Test

File: fvtest/compilertest/README.md [hunk 3]
```diff
@@ -0,0 +1,40 @@
+Test Compiler
+=============
+
+This file contains the OMR 'test compiler', a framework for building language
+independent compiler tests for the Testarossa compiler technology. 
+
+Building
+--------
+
+To Build the test compiler, typing `make` in this directory
```
Comment (0xdaryl) [inline fvtest/compilertest/README.md]: * Build -> build
* test -> test

**Comment by mgaudet** [comment]
Changes made.

**Pull request merged**